x/auction/types/params.go

```diff
@@ -51,3 +74,39 @@ func validatePriceMaxBlockAge(i interface{}) error {
 
 	return nil
 }
+
+func validateMinimumBidInUsomm(i interface{}) error {
+	_, ok := i.(uint64)
+	if !ok {
+		return sdkerrors.Wrapf(ErrMinimumBidParam, "invalid minimum bid in usomm parameter type: %T", i)
+	}
+
+	return nil
+}
+
+func validateAuctionMaxBlockAge(i interface{}) error {
+	auctionMaxBlockAge, ok := i.(uint64)
+	if !ok {
+		return sdkerrors.Wrapf(ErrInvalidAuctionMaxBlockAgeParam, "invalid auction max block age parameter type: %T", i)
+	}
+
+	if auctionMaxBlockAge == 0 {
+		return sdkerrors.Wrapf(ErrInvalidAuctionMaxBlockAgeParam, "blocks to not prune must be non-zero")
```

Comment:
This error message still refers to the old name.

Reply:
Yup, missed this one, ended up removing the 0 check though so this is gone.